src/cow-react/modules/fortune/containers/FortuneWidget/index.tsx

```diff
@@ -1,156 +1,321 @@
     }
     50% {
-      transform: rotate(-5deg) translateY(30px);
+      transform: scale(1) rotate(-5deg)
+    }
+    52% {
+      transform: scale(1.2) rotate(-5deg)
+    }
+    54% {
+      transform: scale(1) rotate(-5deg)
+    }
+    56% {
+      transform: scale(1.2) rotate(-5deg)
+    }
+    58% {
+      transform: scale(1) rotate(-5deg)
     }
     100% {
-      transform: rotate(10deg) translateY(0px);
+      transform: scale(1) rotate(10deg)
     }
   }
-
-  display: inline-block;
-  position: fixed;
-  z-index: 500;
-  right: 100px;
-  bottom: 100px;
-  width: 50px;
-  height: 50px;
-  border-radius: 50%;
-  text-align: center;
-  animation: glowing 1s ease-in-out infinite alternate, swinging 3.5s ease-in-out forwards infinite;
-  cursor: pointer;
-
-  &:before {
-    content: '🎉';
-    font-size: 76px;
-    position: relative;
-    top: -27px;
-    left: -7px;
-  }
+}
 `
 
 const FortuneBanner = styled.div`
   position: fixed;
-  right: 50px;
-  bottom: 20%;
+  width: 100vw;
+  height: 100vh;
+  right: 0;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  margin: auto;
   z-index: 501;
-  background: #fff;
-  padding: 20px;
-  box-shadow: 0 0 20px #fff, 0 0 30px #ff4da6, 0 0 40px #ff4da6, 0 0 50px #ff4da6, 0 0 60px #ff4da6, 0 0 70px #ff4da6,
-    0 0 80px #ff4da6;
-  border-radius: 12px;
+  background: ${({ theme }) => transparentize(0.95, theme.text3)};
+  backdrop-filter: blur(45px);
+  padding: 32px;
+  animation: open 0.3s ease-in-out forwards;
+  overflow-y: auto;
+
+  @keyframes open {
+    from {
+      transform: scale(0);
+    }
+    to {
+      transform: scale(1);
+    }
+  }
 `
 
 const FortuneBannerActions = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  > a {
+    text-decoration: none !important;
+  }
+
+  // Tweet button
+  > a > div {
+    margin: 0;
+    gap: 12px;
+  }
+`
+
+const DontShowAgainBox = styled.div`
+  text-align: center;
+  margin-top: 30px;
+
+  > label {
+    cursor: pointer;
+    display: inline-flex;
+    align-items: center;
+  }
+
+  > label input {
+    width: 18px;
+    height: 18px;
+    margin-right: 8px;
+  }
+`
+
+const FortuneTitle = styled.h2`
   display: block;
+  width: 100%;
+  font-size: 21px;
+  text-align: center;
+  font-weight: 700;
+  color: ${({ theme }) => theme.text2};
+
+  > i {
+    font-size: 16px;
+    text-transform: uppercase;
+    display: block;
+    width: 100%;
+    font-weight: 300;
+    letter-spacing: 2px;
+  }
 `
 
-const FortuneText = styled.h3`
-  margin-top: 0;
-  padding: 20px;
-  font-size: 25px;
-  max-width: 400px;
-  border-radius: 40px;
+const FortuneText = styled.h3<{ isNewFortuneOpen: boolean }>`
+  padding: 24px;
+  width: 100%;
+  font-size: 34px;
+  border-radius: 42px;
+  word-break: break-word;
+  margin: 34px auto 70px;
+  font-weight: 700;
+  text-align: center;
   position: relative;
-  background: rgba(255, 77, 166, 0.09);
+  background: ${({ theme }) => theme.grey1};
+  opacity: ${({ isNewFortuneOpen }) => (isNewFortuneOpen ? 1 : 0.5)};
```

Comment:
@fairlighteth we also need styles, when a fortune is open not for the first time (isNewFortuneOpen === false)